### PR TITLE
SCMOD-10783: Update release notes

### PR DESCRIPTION
**Pull request by rahul-kulkarni73**
https://portal.digitalsafe.net/browse/SCMOD-10783

### CODE REVIEW

File: release-notes-4.1.0.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
 #### Version Number
 ${version-number}
 
 #### New Features
+ - **SCMOD-10189**:  Updated Tomcat base image.  
+  This has changes to meet the logging mandate by adding caf logging Tomcat juli project to Tomcat Java 11 image. 
+
+#### Bug Fixes
+ - **SCMOD-9495**: Adding the REQUIRED_DOCKER_HOST_TYPE=PRIVILEGED setting to the official-build.props file.
```
Comment (dermot-hardy) [inline release-notes-4.1.0.md]: Drop this.  This is not fixing a bug that was present in a previously released version.

**Review by dermot-hardy — DISMISSED** [review]

**Comment by michael-bryson** [comment]
I thought the only fix in this version was the fix for the logging issue in the tomcat image, this should be a patch release.

File: release-notes-4.1.0.md (hunk 1 above)
Comment (rahul-kulkarni73) [reply release-notes-4.1.0.md]: you mean SCMOD-9495 ?

Comment (rahul-kulkarni73) [reply release-notes-4.1.0.md]: Sorry didn't get what you are saying?

Comment (rahul-kulkarni73) [reply release-notes-4.1.0.md]: Sorry didn't get what you are saying?

Comment (dermot-hardy) [reply release-notes-4.1.0.md]: Yes that shouldn't be mentioned in any release notes as it is a change to how the product is built as opposed to being a change to the product.  Think of it like this - downstream consumers are totally unaffected by it.

Comment (rahul-kulkarni73) [reply release-notes-4.1.0.md]: ok got it

File: release-notes-4.1.0.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
+ - **SCMOD-10189**:  Updated Tomcat base image.  
+  This has changes to meet the logging mandate by adding caf logging Tomcat juli project to Tomcat Java 11 image.
```
Comment (michael-bryson) [inline release-notes-4.1.0.md]: Was the tomcat-juli project not in here a couple of releases ago?

Comment (rahul-kulkarni73) [reply release-notes-4.1.0.md]: I looked at this commit https://github.com/CAFapi/opensuse-tomcat-cifs-image/commit/9e50cda84ffec330ddcdd6e7bd001c8658e0d5b8.

Comment (michael-bryson) [reply release-notes-4.1.0.md]: But that version update had nothing to do with adding the juli project, it was fixing the logging issue. Look at the tomcat-jre11-image release notes and just copy that. Also the version of this project needs changed to a patch.

**Review by michael-bryson — APPROVED** [review]

**Comment by dermot-hardy** [comment]
If there are only bug fixes then the version number still needs to change.

**Comment by dermot-hardy** [comment]
@rahul-kulkarni73 This should be 4.0.1 not 4.1.1.

**Comment by rahul-kulkarni73** [comment]
Sorry about that. Fixed now.

**Comment by rahul-kulkarni73** [comment]
testing notification

**Pull request merged**